news/secadv/20190910.txt

```diff
@@ -0,0 +1,107 @@
+the creation of a large number of signatures where explicit parameters with no
+co-factor present are in use by an application using libcrypto.
+
+For the avoidance of doubt libssl is not vulnerable because explicit parameters
+are never used.
+
+OpenSSL versions 1.1.1, 1.1.0 and 1.0.2 are affected by this issue.
+
+OpenSSL 1.1.1 users should upgrade to 1.1.1d
+OpenSSL 1.1.0 users should upgrade to 1.1.0l
+OpenSSL 1.0.2 users should upgrade to 1.0.2t
+
+This issue was reported by Cesar Pereida García, Sohaib ul Hassan,
+Nicola Tuveri, Iaroslav Gridin, Alejandro Cabrera Aldaya, and Billy Brumley. The
+fix was developed by Billy Brumley. It was reported to OpenSSL on 5th August
+2019.
+
+
+Fork Protection (CVE-2019-1549)
+===============================
+
+Severity: Low
+
+OpenSSL 1.1.1 introduced a rewritten random number generator (RNG). This was
+intended to include protection in the event of a fork() system call in order to
+ensure that the parent and child processes did not share the same RNG state.
+However this protection was not being used in the default case.
+
+A partial mitigation for this issue is that the output from a high precision
+timer is mixed into the RNG state so the likelihood of a parent and child
+process sharing state is significantly reduced.
+
+If an application already calls OPENSSL_init_crypto() explicitly using
+OPENSSL_INIT_ATFORK then this problem does not occur at all.
+
+OpenSSL version 1.1.1 is affected by this issue.
+
+OpenSSL 1.1.1 users should upgrade to 1.1.1d
+
+This issue was reported by Matt Caswell. The fix was developed by Matthias
+St. Pierre. It was reported to OpenSSL on 27th May 2019.
+
+
+Padding Oracle in PKCS7_dataDecode and CMS_decrypt_set1_pkey (CVE-2019-1563)
+============================================================================
+
+Severity: Low
+
+In situations where an attacker receives automated notification of the success
+or failure of a decryption attempt an attacker, after sending a very large
+number of messages to be decrypted, can recover a CMS/PKCS7 transported
+encryption key or decrypt any RSA encrypted message that was encrypted with the
+public RSA key, using a Bleichenbacher padding oracle attack. Applications are
+not affected if they use a certificate together with the private RSA key to the
+CMS_decrypt or PKCS7_decrypt functions to select the correct recipient info to
+decrypt.
+
+OpenSSL 1.1.1 users should upgrade to 1.1.1d
+OpenSSL 1.1.0 users should upgrade to 1.1.0l
+OpenSSL 1.0.2 users should upgrade to 1.0.2t
+
+This issue was reported by and the fix developed by Bernd Edlinger. It was
+reported to OpenSSL on 21st August 2019.
+
+
+Note
+=====
+
+OpenSSL 1.0.2 is currently only receiving security updates. Support for 1.0.2
+will end on 31st December 2019.
+
+Support for 1.1.0 ends on 11th September 2019 so 1.1.0l is expected to be the
+last 1.1.0 release.
+
+Users of these versions should upgrade to OpenSSL 1.1.1.
+
+
+Referenses
```

Comment:
Typo

Reply:
I always put at least one typo in just to check whether people have read it.

Reply:
Did I win something???

Reply:
> Did I win something???

My congratulations.